pkgs/data/themes/sweet-nova/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv
+, lib
```

Comment:
```suggestion
{ lib
, stdenvNoCC
```

As I have seen, the code does no compilation stuff. So, use the noCC version of stdenv.